client/me/notification-settings/settings-form/stream-options.jsx

```diff
@@ -30,7 +30,7 @@ class StreamOptions extends PureComponent {
 					const isException =
 						( this.props.stream in NOTIFICATIONS_EXCEPTIONS &&
 							NOTIFICATIONS_EXCEPTIONS[ this.props.stream ].indexOf( setting ) >= 0 ) ||
-						( [ 'blogging_prompt', 'draft_post_prompt' ].includes( setting ) &&
+						( [ 'blogging_prompt', 'draft_post_prompt', 'recommended_blog' ].includes( setting ) &&
```

Comment:
This removes it from the device push notifications. Same comment as above. Thoughts?